src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.sh

```diff
@@ -109,9 +109,14 @@ cp $__TOOLRUNTIME_DIR/runtimes/any/native/* $__TOOLRUNTIME_DIR/
 cp ${__TOOLRUNTIME_DIR}/csc.dll ${__TOOLRUNTIME_DIR}/csc.exe
 cp ${__TOOLRUNTIME_DIR}/vbc.dll ${__TOOLRUNTIME_DIR}/vbc.exe
 
+#Copy RID specific assets to the tools dir since we don't have a deps.json for .NETCore msbuild
+cp ${__TOOLRUNTIME_DIR}/runtimes/unix/lib/netstandard1.3/*.dll $__TOOLRUNTIME_DIR/
```

Comment:
If you published for the current RID you wouldn't need to do this.  Do we have the RID at this point?

Reply:
I don't think that we know the publish RID at this point, which is why we publish only using the framework but not the RID. That said, the only library that really is RID specific out of all of the things we publish is System.Text.Encoding.CodePages.dll which only has a Win and Unix implementation, so I guess that we could just use unix-x64 everywhere and that should probably work.

Reply:
What about using the portable rid (win|linux|osx-<arch>) as well as letting it be specified?  Make sure selfcontained isn't set to true (so that you don't get another copy of the shared framework).

Reply:
@joperezr can you file a tracking issue for this as I agree we should consider other options then making this assumption but for now I want to make forward progress on the SDK update.

Reply:
#1697